SPECS/tg3-module-alt.spec

```diff
@@ -0,0 +1,62 @@
+%define vendor_name Broadcom
+%define vendor_label broadcom
+%define driver_name tg3
+%define module_dir override
+
+%define _version 3.139j
```

Comment:
Is there any use for this `%define`, rather than defining the version directly in the `Version:` tag?